gtsam/navigation/CombinedImuFactor.cpp

```diff
@@ -72,6 +72,17 @@ bool PreintegratedCombinedMeasurements::equals(
 
 //------------------------------------------------------------------------------
 void PreintegratedCombinedMeasurements::resetIntegration() {
+  // Set the initial bias covariance to
+  // the estimated covariance from the last step.
+  p().biasAccOmegaInt = preintMeasCov_.block<6, 6>(9, 9);
+  PreintegrationType::resetIntegration();
+  preintMeasCov_.setZero();
+}
+
+//------------------------------------------------------------------------------
+void PreintegratedCombinedMeasurements::resetIntegration(
+    const gtsam::Matrix6& Q_init) {
+  p().biasAccOmegaInt = Q_init;
```

Comment:
Shouldn't this line go after the one below it? Otherwise biasAccOmegaInt just gets reset again according to your changes to resetIntegration() ?

Reply:
I looked at this again. It's actually correct since it is calling `PreintegrationType::resetIntegration` aka the `resetIntegration` of the base class. I'll still make this change since it will make the code easier to read.

Reply:
Also `PreintegrationType::resetIntegration` resets the `PreintegratedMeasurements` and not the preintegration params which is where the covariances are stored.